.github/workflows/build.yml

```diff
@@ -0,0 +1,102 @@
+name: moonlight-android
```

Comment:
```suggestion
---
name: Build
```

Yaml files should start with `---` and renamed workflow to what it's doing rather the repo it belongs to.

Reply:
Added the changes.